projects/BFDR.CLI/Program.cs

```diff
@@ -646,5 +659,17 @@ private static string Truncate(string value, int length)
                 return value.Substring(0, length);
             }
         }
+
+         private static string Truncate(string value, int length)
```

Comment:
is it possible to call the Truncate defined in VitalRecord/IJE.cs instead of creating another helper function here?

Reply:
I can make the truncate method in IJE public and just refer to that. I feel like putting it into VR/VitalRecord_util is cleaner, no?